docs/data/rpc/rpc-providers.mdx

```diff
@@ -18,16 +18,11 @@ These providers allow access to the Testnet and Mainnet networks.
 | [Gateway](https://gateway.fm/) | [Access](https://gateway.fm/public-rpc/) | Not available | Available | Available |
 | [Ankr](https://www.ankr.com/rpc/advanced-api/) | [Access](https://www.ankr.com/rpc/advanced-api/) | Not available | Available | Available |
 | [Infstones](https://infstones.com/) | [Access](https://cloud.infstones.com/login) | Not available | Not Available | Available |
+| [Liquify](https://www.liquify.io/) | [Access](https://www.liquify.io/) | Available | Available | Available |
```

Comment:
Maybe link out to the mainnet, testnet and futurenet urls on each `Available` word

Reply:
We could! If we put the RPC endpoint URLs and people click on the link it does error out. Is that confusing or normal?
Example:
[](https://stellar-mainnet.liquify.com/api=41EEWAH79Y5OCGI7/mainnet)